http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/http/api/LivenessRoutesSpec.scala

```diff
@@ -8,14 +8,16 @@ import org.broadinstitute.dsde.workbench.leonardo.db.TestComponent
 import org.scalatest.concurrent.Eventually.eventually
 import org.scalatest.flatspec.AnyFlatSpec
 import org.scalatest.matchers.should.Matchers
+import org.scalatestplus.mockito.MockitoSugar
 
 class LivenessRoutesSpec
     extends AnyFlatSpec
     with Matchers
     with ScalatestRouteTest
     with LeonardoTestSuite
     with TestComponent
-    with TestLeoRoutes {
+    with TestLeoRoutes
+    with MockitoSugar {
```

Comment:
I have seen this so many times by now, i really want a mojito!

Reply:
ooh same....maybe a state park next time